### PR TITLE
{Resource} Remove duplicate help messages

### PR DESCRIPTION
**Pull request by Jing-song**
**Related command**

`az account management-group create/update/delete/list/show`
`az account management-group subscription add/show/remove`
`az account management-group tenant-backfill get/start`
`az account management-group entities list`
`az account management-group subscription show-sub-under-mg`
`az account management-group check-name-availability`
`az account management-group hierarchy-settings create/update/list/delete`

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Resource

**Comment by jiasli** [comment]
https://github.com/Azure/azure-cli/blob/e485117c0dadcc15aa909c4651d37edd19794020/src/azure-cli/azure/cli/command_modules/resource/_help.py#L147-L150

**Comment by jiasli** [comment]
One more duplicate `long-summary`:

https://github.com/Azure/azure-cli/blob/e485117c0dadcc15aa909c4651d37edd19794020/src/azure-cli/azure/cli/command_modules/resource/_help.py#L2733-L2737

**Comment by jiasli** [comment]
Those duplicate `long-summary`s were introduced by https://github.com/Azure/azure-cli/pull/6426, https://github.com/Azure/azure-cli/pull/20942.

File: src/azure-cli/azure/cli/command_modules/resource/_help.py [hunk 1]
```diff
@@ -146,8 +141,7 @@
 
 helps['account management-group list'] = """
 type: command
-short-summary: List all management groups.
-long-summary: List of all management groups in the current tenant.
+short-summary: List of all management groups in the current tenant.
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/resource/_help.py:144]:
```suggestion
short-summary: List all management groups in the current tenant.
```

Comment (Jing-song) [reply src/azure-cli/azure/cli/command_modules/resource/_help.py]: it may be misused as a noun

**Review by jiasli — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**